go-controller/pkg/util/multi_network_test.go

```diff
@@ -603,6 +606,103 @@ func TestIsDefault(t *testing.T) {
 	}
 }
 
+func TestGetPodNADToNetworkMapping(t *testing.T) {
```

Comment:
Now that we are at it can we add a commit testing the `WithActiveNetwork` test too ?

Reply:
I mean... we can, but that means that ... you can also.

We can work on top of this PR in any way we want. 

My goal was to provide unit test coverage to this function which I keep forgetting how it works and assuming how I'd want it to work instead.

Reply:
fair enough